cybershell/adaptive/train.py

```
@@ -0,0 +1,49 @@
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report, multilabel_confusion_matrix
+from .mapper import MLMapper, FAMILIES
+@dataclass
+class Sample:
+    text: str
+    labels: Dict[str, int]
+def load_nvd_json_folder(folder: str) -> List[Sample]:
+    samples: List[Sample] = []
+    for path in sorted(glob.glob(os.path.join(folder, "nvdcve-1.1-*.json"))):
+        with open(path, 'r', encoding='utf-8') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)